pkg/process_manager_test.go

```diff
@@ -107,3 +110,73 @@ func innerStartWaitStatus(t *testing.T, pm *ProcessManager) {
 	assert.Equal(EXITED, st.State)
 	assert.Equal(1, st.ExitCode)
 }
+
+func TestStream(t *testing.T) {
+	assert := assert.New(t)
+
+	pm := new(ProcessManager)
+	defer pm.StopAll()
+
+	// Ideally this test would use an IPC mechanism for synchronization, as time
+	// is not a synchronization primitive, and relying on timing for
+	// synchronization causes tests to become unnecessarily slow and unreliable.
+
+	id, err := pm.Start("bash", "bash", "-c", `
+		for x in $(seq 1 3); do
+			sleep 0.25
+			echo "test $x"
+		done
+	`)
+	if !assert.Nil(err) {
+		return
+	}
```

Comment:
informational nit, but testify's `require` package will do this for you:
```suggestion
	require.NoError(t, err)
```

(linters like [`testifylint`](https://github.com/Antonboom/testifylint) will complain about this)

Reply:
Oh that's useful! I had missed the docs link badge in testify's readme and only read the readme itself so I didn't realize this existed...